site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -751,6 +802,34 @@ include::../shared/_premium-edition-blurb-api.adoc[]
 +
 :premium_feature!:
 
+[field]#tenant.scimServerConfiguration.clientEntityTypeId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.36.0#::
+The Entity Type that will be used to represent SCIM Clients for this tenant.
++
+:enterprise_feature: SCIM
+include::../shared/_enterprise-edition-blurb-api.adoc[]
++
+Required when [field]#tenant.scimServerConfiguration.enabled# is `true`.
+
+[field]#tenant.scimServerConfiguration.enabled# [type]#[Boolean]# [optional]#Optional# [since]#Available since 1.36.0#::
+Whether or not this tenant has the SCIM endpoints enabled.
++
+:enterprise_feature: SCIM
+include::../shared/_enterprise-edition-blurb-api.adoc[]
+
+[field]#tenant.scimServerConfiguration.schemas# [type]#[Map]# [optional]#Optional# [since]#Available since 1.36.0#::
+JSON formatted as a SCIM Schemas endpoint response. Because the SCIM lambdas may modify the JSON response, ensure the Schema's response matches that generated by the response lambdas.
```

Comment:
Maybe add a link to the SCIM Schemas schema definition here.

https://datatracker.ietf.org/doc/html/rfc7643#section-7